### PR TITLE
add contact details info as optional step for immersve conducted kyc

### PR DESCRIPTION
**Pull request by forktrucka**
Adds the contract detail info as optional step for IC KYC.

Ticket Link: https://immersve.slack.com/archives/C06MGUX55L6/p1729194615408009

### CODE REVIEW

File: imsv-docs-astro/src/content/docs/guides/kyc/immersve-conducted-kyc.mdoc [hunk 1]
```diff
@@ -87,7 +87,44 @@ curl -X POST "https://test.immersve.com/api/spending-prerequisites" \
   ]
 }
 ```
+### (Optional) Supply Contact Details
```
Comment (NicBathgate) [inline imsv-docs-astro/src/content/docs/guides/kyc/immersve-conducted-kyc.mdoc:90]: This looks good! Only feedback is, I wonder if we should have (Optional) at the end of the title, like Supply Contact Details (Optional), or maybe even mention in the first paragraph that it's optional?

Comment (forktrucka) [reply imsv-docs-astro/src/content/docs/guides/kyc/immersve-conducted-kyc.mdoc:90]: at the end sure. i figure most dev's skim things, so making sure it's clear that is an optional step in the heading seemed to make the most sense.

Comment (forktrucka) [inline imsv-docs-astro/src/content/docs/guides/kyc/immersve-conducted-kyc.mdoc:90]:
```suggestion
### Supply Contact Details (Optional)
```

**Review by NicBathgate — APPROVED** [review]
Nice!

**Pull request merged**